python_utils.py

```diff
@@ -431,3 +431,8 @@ def yaml_from_dict(dictionary, width=80):
     """
     dictionary = _recursively_convert_to_str(dictionary)
     return yaml.safe_dump(dictionary, default_flow_style=False, width=width)
+
+
+def abort():
```

Comment:
I think there should already be a native assert that checks for exception. I’d recommend grep-ing for those.

Reply:
Done! I followed this [link](https://stackoverflow.com/questions/129507/how-do-you-test-that-a-python-function-throws-an-exception).